README.md

```diff
@@ -56,35 +56,23 @@ site {
 
 ## Reference
 
-### Application Cloudshop
-
-* `dbinstance`
-  + The name of the database instance for MS SQL Server
-* `dbpassword`
-  + The password to use for your MS SQL database instance
-* `dbuser`
-  + The user to log into your MS SQL database instance
-* `dbname`
-  + The name of your MS SQL database
-* `dbport`
-  + The port that MS SQL Server is listening on
-* `iss_site`
-  + The directory that your Cloudshop module should go under in IIS
-* `docroot`
-  + The directory on disk where your website should be hosted from
-* `file_source`
-  + The location for obtaining either MS SQL Server iso or the Cloudshop application archive
-* `administrator`
-  + The user that should be used to install MS Sql
-* `app_count`
-  + How many Cloudshop application instances you wish to set up
+### Application CloudShop
+
+* `dbinstance`: The name of the database instance for the Microsoft SQL Server instance.
+* `dbpassword`: The password to use for the Microsoft SQL Server instance.
+* `dbuser`: The user to log into the Microsoft SQL Server instance.
+* `dbname`: The name of the Microsoft SQL Server instance.
+* `dbport`: The port the Microsoft SQL Server listens on.
+* `iss_site`: The IIS directory that contains the CloudShop module.
+* `docroot`: The directory on disk from which your website is hosted.
+* `file_source`: The location for obtaining the Microsoft SQL Server ISO or the location of the CloudShop application archive.
+* `administrator`: The user used to install the Microsoft SQL Server.
+* `app_count`: The number of CloudShop application instances to set up.
 
 ## Limitations
 
 ### Known Issues
 
-https://tickets.puppetlabs.com/browse/ORCH-1285
```

Comment:
I think this is fixed in the latest pxp-agent so probably we can just delete it.